includes/modules/bootstrap/product_listing.php

```diff
@@ -150,139 +210,221 @@
+//                        $lc_text .= '<div class="list-image">';
+                        $lc_text .= '<a href="' . $href . '">';
+                        $lc_text .= zen_image(DIR_WS_IMAGES . $record['products_image'], $listing_product_name, IMAGE_PRODUCT_LISTING_WIDTH, IMAGE_PRODUCT_LISTING_HEIGHT, 'class="img-fluid listingProductImage"');
+                        $lc_text .= '</a>';
+//                        $lc_text .= '</div>';
                     }
                     break;
             }
 
-            $product_contents[] = $lc_text; // (used in column mode)
+            $product_contents[] = $lc_text; // (used in column/fluid modes)
 
-            if ($product_listing_layout_style == 'rows') {
+            if ($product_listing_layout_style === 'table') {
                 $align_class = empty($lc_align) ? '' : " text-$lc_align";
                 $list_box_contents[$rows][] = [
                     //'align' => $lc_align, // not used with Bootstrap template: converted to css class on next line
                     'params' => 'class="productListing-data' . $align_class . '"',
-                    'text'  => $lc_text,
+                    'category' => $record['master_categories_id'],
+                    'parent_category_name' => $record['parent_category_name'],
+                    'category_name' => $record['category_name'],
+                    'manufacturers_id' => $record['manufacturers_id'],
+                    'manufacturers_name' => $record['manufacturers_name'],
+                    'text' => $lc_text,
                 ];
-//        // add description and match alternating colors
-//        if (PRODUCT_LIST_DESCRIPTION > 0) {
-//          $rows++;
-//          if ($extra_row == 1) {
-//            $list_box_description = "productListing-data-description-even";
-//            $extra_row=0;
-//          } else {
-//            $list_box_description = "productListing-data-description-odd";
-//            $extra_row=1;
-//          }
-//          $list_box_contents[$rows][] = array('params' => 'class="' . $list_box_description . '" colspan="' . $zc_col_count_description . '"',
-//                                              'text' => zen_trunc_string(zen_clean_html(stripslashes(zen_get_products_description($record['products_id'], $_SESSION['languages_id']))), PRODUCT_LIST_DESCRIPTION));
-//        }
+//                // add description
+//                if (!empty($listing_description)) {
+//                    $rows++;
+//                    // match alternating colors
+//                    if ($extra_row == 1) {
+//                        $tmp_class_name = "productListing-data-description-even";
```

Comment:
Again, just consistency with core ZC module file.

I was aiming to keep the file in sync with core code so that it's easy to identify customizations one has made, in order to know what styling is unique to a particular edition of the file (eg template).